refactor: update proguard documentation (#326)
ZupIT/docs-beagle: content/en/resources/customization/beagle-for-android/proguard-configuration.md

## Introduction

Beagle contains minimum rules necessary for its components to work without the need for your application to add any rules in ProGuard / R8. However in the annotations `RegisterWidget` and ` RegisterAction` you need to fill them with their respective names and if you have an object inside your component you need to annotate it with `@ BeagleJson`

## Example

> To ensure that your component works normally with ProGuard / R8 active, you should note the classes used with the annotation `@BeagleJson:`

Reviewer: ```suggestion
To ensure that your component works normally with ProGuard / R8 active, you must annotate the classes used with the annotation `@BeagleJson:`
```
